test/parallel/test-http-exceptions.js

```diff
@@ -20,8 +20,13 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 'use strict';
-require('../common');
+const common = require('../common');
+const Countdown = require('../common/countdown');
 const http = require('http');
+const NUMBER_OF_EXCEPTIONS = 4;
+const countdown = new Countdown(NUMBER_OF_EXCEPTIONS, () => {
+  process.exit(0);
```

Comment:
Does `server.close()` not do the trick here?

Reply:
it does not, probably because of the intended exception throwing `intentionally_not_defined`. it keeps `res` from being closed with `.end();`. My code only refactors the `common.mustCall` from the countdowns.